src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx

```diff
@@ -4,7 +4,7 @@ import { action, computed, observable, makeObservable } from 'mobx';
 import autobind from 'autobind-decorator';
```

Comment:
Change name of this file/component becaues it's not just the plot.   Wrapper.  Content